critiquebrainz/db/review.py

```diff
@@ -684,3 +684,28 @@ def reviewed_entities(*, entity_ids, entity_type):
         })
         reviewed_ids = [str(row[0]) for row in results.fetchall()]
     return reviewed_ids
+
+def get_top_contributors(limit=10):
+    query = sqlalchemy.text("""
+        SELECT review.user_id,
+            COUNT(*) as qtd,
+            "user".email,
+            "user".created as user_created,
+            "user".display_name,
+            "user".show_gravatar,
+            "user".musicbrainz_id,
+            "user".is_blocked
+        FROM review JOIN "user" 
+        ON review.user_id = "user".id
+        WHERE review.published_on > current_date - interval '7 days'
+        AND review.is_draft <> TRUE
+        GROUP BY review.user_id, "user".id
+        ORDER BY qtd DESC
+        LIMIT :limit
+    """)
+    with db.engine.connect() as connection:
+        results = connection.execute(query, {"limit": limit})
+        rows = results.fetchall()
+        return rows
+
```

Comment:
Again here I guess you should not be directly returning the results.
It's a good programming practice to consider cases like:
What happens if db is not able to fetch any result?
What is the `type` of result? Do we need to convert results to some suitable `type` before returning them?
And so on.

Reply:
I understand, i had not stopped to think about what you said, but it makes sense